package.json

```diff
@@ -59,9 +59,9 @@
       "prettier --cache --ignore-path .gitignore --ignore-unknown --write"
     ]
   },
-  "packageManager": "pnpm@8.15.8",
+  "packageManager": "pnpm@9.1.4+sha512.9df9cf27c91715646c7d675d1c9c8e41f6fce88246f1318c1aa6a1ed1aeb3c4f032fcdf4ba63cc69c4fe6d634279176b5358727d8f2cc1e65b65f43ce2f8bfb0",
```

Comment:
the hash is generated by `corepack up` , you can look it up manually with 

```
curl `npm info pnpm@9.1.4 dist.tarball` | sha512sum
```

it's a bit unfortunate that corepack went that way of attaching a checksum of the tgz, but better than nothing.
I don't use corepack but for those who do, they recommend having the hash for checking purposes.